client/sites-dashboard-v2/sites-dataviews/index.tsx

```diff
@@ -113,13 +113,8 @@ const DotcomSitesDataViews = ( {
 				id: 'status',
 				header: __( 'Status' ),
 				render: ( { item }: { item: SiteInfo } ) => <SiteStatus site={ item } />,
-				type: 'enumeration',
-				elements: siteStatusGroups,
-				filterBy: {
-					operators: [ 'in' ],
```

Comment:
I assume we can also remove https://github.com/Automattic/wp-calypso/blob/trunk/client/sites-dashboard-v2/sites-dashboard.tsx#L102-L111